src/js/formapp.js

```diff
@@ -172,6 +172,12 @@ const Router = Backbone.Router.extend({
       this.trigger(data.message, data.args);
     }, false);
 
+    const self = this;
+
+    $(document).on('pointerdown', function(evt) {
+      self.request('user-activity', { eventType: 'body:down' });
+    });
```

Comment:
I attempted to do this here: 

```js
$(document).on('pointerdown', function(evt) {
    self.request('user-activity', { eventType: 'body:down', evt });
});
```

But was getting this error:

```js
formapp.js:249 Uncaught (in promise) DOMException: Failed to execute 'postMessage' on 'Window': PointerEvent object could not be cloned.
```

And couldn't figure out why it was happening or how to fix it.

Reply:
you can only send serializeable things over postMessage... so things that can become text and get parsed on the other.. pure json objects, but nothing more complicated or that has a reference to window or something.